.travis.yml

```diff
@@ -1,9 +1,12 @@
 sudo: false
 language: node_js
 node_js:
-  - '0.12'
+  - '4.6.2'
```

Comment:
@Axl-Nolasco where did these changes came from?
Is is something with your dev environment?